taglib/asf/asftag.h

```diff
@@ -101,48 +101,48 @@ namespace TagLib {
       /*!
        * Sets the title to \a s.
        */
-      void setTitle(const String &s) override;
+      void setTitle(const String &value) override;
 
```

Comment:
The documentation comment refers to `s` which is the old variable name.

Reply:
that doesn't look like a proper Doxygen comment.

Reply:
I'm not sure what constitutes proper vs. improper doxygen comments but the variable names need to match.

Reply:
ah nvm. Seems it is. I'll fix them. Oddly enough clang warns only on \param and not \a.